test/functional/test_vm_default.py

```diff
@@ -6,7 +6,7 @@
     vm status: running
     vm cpu count: 4
     vm disk count: 1
-    vm disk size: 16G
+    vm disk size: 4G
```

Comment:
yes, quanta_d51 is the default node type
by default, the disk capacity is 4G. because i changed the size to 16G before, the image size is still 16G even restart the VM. this is a code bug after confirming with @MarkMa001